docs/README.md

```diff
@@ -20,21 +20,23 @@ The LogDNA agent is a fast, resource-efficient log collection client that forwar
   * [Running as Non-Root](#running-as-non-root)
```

Comment:
We should take this out for 2.2 GA Docs.

Reply:
They'll disappear when I merge back to the 2.2 branch